app/assets/stylesheets/sass/buttons.scss

```diff
@@ -0,0 +1,45 @@
+.tiny {
+  padding: 0px;
+}
+
+@mixin buttonstyles($bg-color: #bbb) {
```

Comment:
Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.